lib/c_glib/src/thrift/c_glib/transport/thrift_socket.h

```diff
@@ -50,11 +50,8 @@ struct _ThriftSocket
 
   /* private */
   gchar *hostname;
-  gshort port;
+  guint port;
   int sd;
-  guint8 *buf;
```

Comment:
This is the default implementation of the socket. Are you sure about it? If this is changed I suppose this should be done in it's own ticket.